config/zephyr/Kconfig

```
@@ -336,7 +336,8 @@ if CHIP_ENABLE_ICD_SUPPORT
 
 config CHIP_ICD_SLOW_POLL_INTERVAL
 	int "Intermittently Connected Device slow polling interval (ms)"
-	default 1000
+	default 1000 if !CHIP_ICD_LIT_SUPPORT
+	default 30000 if CHIP_ICD_LIT_SUPPORT
```

```suggestion
	default 30000 if CHIP_ICD_LIT_SUPPORT
	default 1000
```
First matching default always wins so no need to repeat the negated condition.